en/developer_guide/install_with_vagrant.rst

```diff
@@ -69,8 +69,9 @@ Install open-orchestra with composer:
 
 .. code-block:: bash
 
-    $ ./composer.phar create-project open-orchestra/open-orchestra path/to/your/folder -s dev
-    $ ./composer.phar create-project open-orchestra/open-orchestra-front-demo path/to/your/folder -s dev
+    $ ./composer.phar create-project open-orchestra/open-orchestra path/to/your/back/folder -s dev
+    $ ./composer.phar create-project open-orchestra/open-orchestra-front-demo path/to/your/front/folder -s dev
+    $ ./composer.phar create-project open-orchestra/open-orchestra-provision path/to/your/provisionning/folder -s dev
```

Comment:
No, this doc is not a non sense, the line just after this one explain how you can get the provisionning on your computer.

`composer` is only used to install php packages